### PR TITLE
[SERV-1105] Establish basic files for pt-tools project

### PR DESCRIPTION
**Pull request by angelahuqing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+
+To produce a tar/gzipped output or upack a tar/gzipped in the pairtree structure run 
+
+    ptcp -a [/path/to/ID.tgz] [ID]
+
+This provides a way to archive an item from the pairtree and un-archive it again back into a pairtree structure, but it's not intended as a way to create archives within the pairtree structure. 
+
+## ptmv
+
+Ptmv is a mv-like tool that can move files in and out of the Pairtree structure. Ptmv operates similarly to ptcp except it is destructive, removing the “from” source and overwriting the “to” destination (so deleting the existing directory, if there is one). Ptmv only works on the directory/Pairtree object level and not at the level of files within the Pairtree object, so all sources and targets should represent directories instead of individual files. Ptmv supports the -a option to tar/gzip things being moved between the file system and Pairtree structure.
```
Comment (ksclarke) [inline README.md:51]: Are those quotes around `from` and `to` standard (i.e., `"`) quotes or "fancy" quotes? Could we stick to the standard ones?

**Review by ksclarke — COMMENTED** [review]
I wonder, since we've broken out the arguments, whether there shouldn't be a line for just the command without the/an argument too (so that it's clearer that the arguments are option)?

File: README.md (hunk 1 above)
Comment (ksclarke) [reply README.md:51]: I'm curious is there was a reason you didn't treat `-a` like you did in the above examples (breaking it out from the textual description)?

**Comment by angelahuqing** [comment]
*Note the spec has changed even since this iteration, but I think it would be best to have the basics in the project at this point and I will update the README in future PRs as new decisions are made

**Review by ksclarke — APPROVED** [review]

**Review by DRickard — APPROVED** [review]

**Pull request merged**